libs/pageserver_api/src/models.rs

```diff
@@ -177,6 +177,28 @@ serde_with::serde_conv!(
     |value: String| -> Result<_, humantime::TimestampError> { humantime::parse_rfc3339(&value) }
 );
 
+impl LsnLease {
+    /// The default length for an explicit LSN lease request (10 minutes).
+    #[cfg(not(feature = "testing"))]
```

Comment:
I feel we want to shorten the lease interval for testing. Not sure what are the best practice for using feature flags.

Reply:
This looks fine. I think this is also possible with const fns if you feel that's a better way to do that?

Reply:
I'm actually trying to put these inside `TenantConf` based on Conrad's suggestion so we can have a runtime config? Does that sounds good?